.github/workflows/on-push.yaml

```diff
@@ -0,0 +1,132 @@
+name: Push
+
+env:
+  ACRONYM: coms
+  APP_NAME: common-object-management-service
+  NAMESPACE_PREFIX: bb17f9
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v*.*.*
+
+jobs:
+  build:
+    name: Build & Push to GHCR
+    runs-on: ubuntu-latest
+    outputs:
+      HAS_DOCKERHUB_SECRETS: ${{ steps.check-secrets.outputs.HAS_DOCKERHUB_SECRETS }}
```

Comment:
does this mean the secrets for dockerhub are the same as for ghcr?

Reply:
No - this step here is just doing a pre-run check to detect if the dockerhub secrets are around. If they are in the repo settings, It's used later in line 42 to determine if pushing to additional registries is to run or not.